server/endpoint/creator/endpoint.go

```diff
@@ -99,7 +99,9 @@ func (e *Endpoint) Endpoint() kitendpoint.Endpoint {
 		}
 
 		creatorResponse, err := e.service.Create(ctx, creatorRequest)
-		if err != nil {
+		if creator.IsAlreadyExists(err) {
+			return nil, microerror.Maskf(alreadyExistsError, err.Error())
```

Comment:
Here we use stack information. Not sure if this is the best way to convert error to another package.